openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-dataextractor/src/tools/DataExtractor/DataExtractor.vue

```diff
@@ -932,7 +932,9 @@ export default {
           let columnIndex = columnMap[key]
           // Get the value and put it into the correct column
           if (typeof packet[key] === 'object') {
-            if (Array.isArray(packet[key])) {
+            if (packet[key] === null) {
+              row[columnIndex] = ''
```

Comment:
Any idea how we'd be getting null in here? I agree it's good to explicitly handle it.

Reply:
It happens if you playback over items that don't exist in earlier versions of a packet.